packages/bot-web-ui/src/components/dashboard/bot-builder/workspace-wrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import Flyout from 'Components/flyout';
+import Toolbox from './toolbox';
+import Toolbar from './toolbar';
+import './workspace.scss';
+
+type TWorkspaceWrapper = {
+    onMount: () => void;
+    onUnmount: () => void;
+    is_loading: boolean;
+};
+
+const WorkspaceWrapper = ({ onMount, onUnmount, is_loading }: TWorkspaceWrapper) => {
+    React.useEffect(() => {
+        onMount();
+        return () => {
+            onUnmount();
+        };
+    }, []);
+
+    if (is_loading) return null;
+
+    return (
+        <>
+            {Blockly?.derivWorkspace && (
+                <>
```

Comment:
Please use React.Fragment as this is the styleguide in deriv-app.

Reply:
done ✅